proposals/0242-alert-style-subtle.md

```diff
@@ -6,125 +6,203 @@
 
-Places where subtle alerts may be useful include messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presented outside of the current screen-takeover method.
+Subtle Alerts would be especially useful when the app is in the background. Places where subtle alerts may be useful include messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presented outside of the current screen-takeover method.
 
 ## Proposed solution
-The proposed solution is to create a new `AlertStyle` enum that can be sent along with the `Alert` RPC.
+The proposed solution is to create a new RPC `SubtleAlert` that can be sent by developers to show a less prominent kind of Alert.
 
 ### Functionality
-Functionally, this would be identical to the prominent alert style with the following exceptions:
+Functionally, this would be identical to the standard (prominent / modal) `Alert` style with the following exceptions:
 
-1. It may support a different number of text lines or soft buttons.
-2. `progressIndicator` parameter should do nothing.
-3. Touching (or otherwise selecting) the alert should open the app.
-4. Touching outside of the subtle alert should close the alert.
+### MOBILE_API Changes
 
-### MOBILE_API and HMI_API Changes
-
-#### Capabilities
+#### Capabilities Updates
 ```xml
-<struct name="DisplayCapabilities" since="1.0">
-    <!-- All existing params -->
+<enum name="TextFieldName" since="1.0">
+    <!-- New Values -->
+    <element name="subtleAlertText1">
+        <description>The first line of the subtle alert text field; applies to `SubtleAlert` `alertText1`</description>
+    </element>
+    <element name="subtleAlertText2">
+        <description>The second line of the subtle alert text field; applies to `SubtleAlert` `alertText2`</description>
+    </element>
+    <element name="subtleAlertSoftButtonText">
+        <description>A text field in the soft button of a subtle alert; applies to `SubtleAlert` `softButtons`</description>
+    </element>
+</enum>
 
-    <param name="alertStylesAvailable" type="AlertStyle" array="true" minValue="1" maxValue="1000" mandatory="false" since="6.x">
-        <description>An array of available alert styles. If this parameter is not provided, only the `PROMINENT` style is assumed to be available.</description>
+<enum name="ImageFieldName" since="1.0">
+    <!-- New Values -->
+    <element name="subtleAlertIcon">
+        <description>The image of the subtle alert; applies to `SubtleAlert` `alertImage`</description>
+    </element>
+    <element name="subtleAlertSoftButtonImage">
```

Comment:
Does ImageFieldName:softButtonImage not apply for a subtle alert?

Reply:
It probably can be. I can't think of a case where the soft button would be available in the notification and not in the main app.